docs/example-react-intl.md

```diff
@@ -0,0 +1,51 @@
+---
+id: example-react-intl
+title: React Intl
+---
+
+```jsx
+import React from 'react'
+import 'jest-dom/extend-expect'
+import { render, cleanup, getByTestId } from 'react-testing-library'
+import { IntlProvider, FormattedDate } from 'react-intl'
+import IntlPolyfill from 'intl'
+import 'intl/locale-data/jsonp/pt'
+
+const setupTests = () => {
+  // https://formatjs.io/guides/runtime-environments/#server
+  if (global.Intl) {
+    Intl.NumberFormat = IntlPolyfill.NumberFormat
+    Intl.DateTimeFormat = IntlPolyfill.DateTimeFormat
+  } else {
+    global.Intl = IntlPolyfill
+  }
+}
+
+const FormatDateView = () => {
+  return (
+    <div data-testid="date-display">
+      <FormattedDate
+        value="2019-03-11"
+        timeZone="utc"
+        day="2-digit"
+        month="2-digit"
+        year="numeric"
+      />
+    </div>
+  )
+}
+
+const renderWithReactIntl = component => {
+  return {
+    ...render(<IntlProvider locale="pt">{component}</IntlProvider>),
```

Comment:
Nit: do you need the `...` here? I think returning the result of render is fine.

Reply:
You are absolutely correct, it can be removed. I'll update it.

I have a codesandbox with a functional example of it here: https://codesandbox.io/s/yq2jz85jx1